src/ucp/core/ucp_ep.c

```diff
@@ -1113,6 +1113,17 @@ void ucp_ep_config_init(ucp_worker_h worker, ucp_ep_config_t *config)
         }
     }
 
+    /* configuration for min zcopy */
+    config->tag.rndv.min_get_zcopy = 0;
+    for (lane = 0; lane < config->key.num_lanes &&
```

Comment:
let's use 'i' instead of 'lane' , because it's not the lane itself, it's an index into rma_bw_lanes

Reply:
done